CONTRIBUTING.md

```diff
@@ -52,35 +52,36 @@ Once you have a GitHub login:
 
 2. Then start a terminal (or use your favorite git GUI app) and clone your fork of the Exiv2 repo:
 
-        $  git clone https://github.com:your-username/exiv2.git
+        $ git clone https://github.com:your-username/exiv2.git
+        $ cd exiv2
+
+3. Set (add) the remote repository:
```

Comment:
I would say something like this:

```
3. Add a new remote pointing to upstream
```
IMO the current sentence is a bit ambiguous, since the user fork is also "a remote repository"

Reply:
Yes I agree. I will change it to your suggestion.